src/cl/kona-node/launcher.star

```diff
@@ -168,7 +168,15 @@ def get_service_config(
     ]
 
     if observability_helper.enabled:
-        _observability.configure_op_service_metrics(cmd, ports)
+        cmd += [
```

Comment:
is that needed?

It seems pretty similar to the `configure_op_service_metrics` code, except for the separated arguments?